Magento/magento_readme.md

```diff
@@ -0,0 +1,58 @@
+* [Azure Database for MySQL](https://azure.microsoft.com/services/mysql/) stores transaction data such as orders and catalogs. Using version 8.0 is recommended.
+* [Azure Files Premium](https://azure.microsoft.com/services/storage/files/) or an equivalent network-attached storage (NAS) system stores media files such as product images. Magento needs a Kubernetes-compatible file system such as Azure Files Premium or [Azure NetApp Files](https://azure.microsoft.com/services/netapp/), which can mount a volume in *ReadWriteMany* mode.
+* A [content delivery network (CDN)](https://azure.microsoft.com/services/cdn/) serves static content such as CSS, JavaScript, and images. Serving content through a CDN minimizes network latency between users and the datacenter. A CDN can remove significant load from NAS by caching and serving static content.
+* [Redis](https://redis.io/) stores session data. Hosting Redis on containers is recommended for performance reasons.
+* AKS uses an Azure [Active Directory (Azure AD)](https://azure.microsoft.com/services/active-directory/) identity to create and manage other Azure resources like Azure load balancers, user authentication, role-based access control, and managed identity.
+* [Azure Container Registry](https://azure.microsoft.com/services/container-registry/) stores the private [Docker](https://www.docker.com/) images that are deployed to the AKS cluster. You can use other container registries like Docker Hub. Note that the default Magento install writes some secrets to the image.
+* [Azure Monitor](https://azure.microsoft.com/services/monitor/) collects and stores metrics and logs, including Azure service platform metrics and application telemetry. Azure Monitor integrates with AKS to collect controller, node, and container metrics, and container and master node logs.
+
+## How to deploy the ARM template
+
+To deploy this ARM template, open the Azure CLI, and then run the following command:
+
+```azurecli-interactive
+az deployment group create --resource-group <ResourceGroupName> --template-file azuredeploy.json
+```
+
+## Parameters
+
+The following table shows a list of parameters to determine how much the cluster can scale. The default values are intentionally scaled down to avoid unnecessary costs for PoC. Values in the rightmost column are the ones we used in our performance test, when we achieved 340 orders per minute using the [Magento performance toolkit](https://github.com/magento/magento2/tree/2.4/setup/performance-toolkit).
+
+| **Parameter** | **Default value** | **Value in our performance test** |
+|---|---|---|
+| `aksNodePoolSize` | 3 | 10 |
+| `aksNodePoolVMSize` | Standard_DS2_v2 | Standard_F8s_v2 |
+| `mySQLSkuName` | MO_Gen5_4 | MO_Gen5_16 |
+| `mySQLSkuTier` | MemoryOptimized | MemoryOptimized |
+| `mySQLSkuFamily` | Gen5 | Gen5 |
+| `mySQLSkuCapacity` | 4 | 16 |
+| `mySQLStorageSize` | 128000 | 512000 |
+| `fileShareQuota` | 512 | 16384 |
+

```

Comment:
Include a section  like Next Steps. After they deploy and run the app,  they might care about backup or auth with AAD , deploy with DevOps for AKS etc. 2-3 next step links would be useful here

Reply:
Both added to the new PR